app/models.py

```diff
@@ -761,41 +761,7 @@ def framework_names(self):
 
     @property
     def category_name(self):
-        price = ServiceTypePrice.query.filter(ServiceTypePrice.supplier_code == self.code).first()
-        if price:
-            return price.service_type.category.name
-
-    @property
-    def regions(self):
-        prices = ServiceTypePrice.query.\
-            join(ServiceTypePrice.region).\
-            filter(ServiceTypePrice.supplier_code == self.code).\
-            distinct(Region.state, Region.name).\
-            order_by(Region.state, Region.name).\
-            all()
-
-        if prices:
-            return [dict(state=p.region.state, name=p.region.name) for p in prices]
-
-    @property
-    def services(self):
-        service_types = db.session\
-            .query(ServiceTypePrice.service_type_id,
-                   ServiceType.name, ServiceTypePrice.sub_service_id, ServiceSubType.name.label('sub_service_name'))\
-            .join(ServiceType, ServiceTypePrice.service_type_id == ServiceType.id)\
-            .outerjoin(ServiceSubType, ServiceTypePrice.sub_service_id == ServiceSubType.id)\
-            .filter(ServiceTypePrice.supplier_code == self.code)\
-            .group_by(ServiceTypePrice.service_type_id, ServiceType.name,
-                      ServiceTypePrice.sub_service_id, ServiceSubType.name)\
-            .order_by(ServiceType.name)\
-            .all()
-
-        result = []
-        for key, group in groupby(service_types, key=lambda x: dict(id=x.service_type_id, name=x.name)):
-            subcategories = [dict(id=s.sub_service_id, name=s.sub_service_name) for s in group]
-            result.append(dict(key, subCategories=subcategories))
-
-        return result
+        return None
```

Comment:
why is this property not deleted?

Reply:
You're right, this was related to some endpoints for ORAMS.  These have been removed.